src/main/java/thrift/logic/parser/BudgetCommandParser.java

```diff
@@ -0,0 +1,37 @@
+package thrift.logic.parser;
+
+import java.util.Calendar;
+
+import thrift.commons.core.Messages;
+import thrift.logic.commands.BudgetCommand;
+import thrift.logic.parser.exceptions.ParseException;
+import thrift.model.transaction.Budget;
+import thrift.model.transaction.BudgetValue;
+
+/**
+ * Parses input arguments and creates a new BudgetCommand object.
+ */
+public class BudgetCommandParser implements Parser<BudgetCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the BudgetCommand and returns an BudgetCommand
+     * object for execution.
+     * @throws ParseException if the user input does not conform to the expected format.
```

Comment:
There should be an empty line between description and parameter section.

Reply:
Fixed